components/utils/src/fs.rs

```diff
@@ -95,19 +95,23 @@ pub fn find_related_assets(path: &Path) -> Vec<PathBuf> {
 
 /// Copy a file but takes into account where to start the copy as
 /// there might be folders we need to create on the way
-pub fn copy_file(src: &Path, dest: &PathBuf, base_path: &PathBuf) -> Result<()> {
+pub fn copy_file(src: &Path, dest: &PathBuf, base_path: &PathBuf, hardlink: bool) -> Result<()> {
```

Comment:
is it called `hardlink` instead of `hard_link` because of shadowing? You can just use `std::fs::hard_link` in the function body if that's the case instead.

Reply:
Not that much experience with Rust here, thanks for the hint :>